### ISSUE TITLE
Question: How does end-of life for Facebook Graph API v2.2 affect rainloop?

### ISSUE THREAD

**Issue by AndrewSav** [RainLoop/rainloop-webmail#1349]
I just received the following email from the facebook:

```
v2.2 Upgrade Reminder
RainLoop Webmail currently has access to Graph API v2.2 which will reach the end of its 2-year lifetime on 27 March, 2017. To ensure a smooth transition, please migrate all calls to Graph API v2.3 or higher.

To check if your app will be affected by this upgrade you can use the Version Upgrade Tool. This will show you which calls, if any, are affected by this change as well as any replacement calls in newer versions. If you do not see any calls, your app may not be affected by this change.

You can also use our changelog to see the full list of changes in all Graph API versions.

You can view this and other Developer Notifications related to your app, in the App Dashboard.
```

What needs to be done on my mail server, so that RainLoop does not loose FB integration after 27 March, 2017?

**Comment by madpsy** [comment]
Should this not be closed as of 1.11.0?

**Comment by RainLoop** [comment]
Yes it is already fixed in the latest version.
